torch/_inductor/codegen/common.py

```diff
@@ -320,6 +320,10 @@ def _print_GreaterThan(self, expr):
         # Go figure...
         return " >= ".join(map(self.paren, map(self._print, expr.args)))
 
+    def _print_Piecewise(self, expr):
+        # This is a hack, but we only ever emit `Piecewise` when converting bool to int
```

Comment:
I guess the proper way to do this would be to create a `BoolToInteger` function.

Reply:
Since these are all strings we can also just pattern match on the piece wise args (1, val), (0, True)